scripts/check_ipfs_hashes_pushed.py

```diff
@@ -76,9 +78,23 @@ def get_file_from_tag(file_path: str, latest_tag: Optional[str] = None) -> str:
     # Get all hashes from the latest tag, excluding the scaffold ones (that are not pushed)
 
     packages_json = json.loads(get_file_from_tag("packages/packages.json"))
-    errors = {k: v for k, v in packages_json.items() if not check_ipfs_hash_pushed(v)}
-
-    if errors:
-        print(f"The following hashes were not found in IPFS registry: {errors}")
-        sys.exit(1)
-    print("OK")
+    with ThreadPoolExecutor(max_workers=MAX_WORKERS) as executor:
+        futures = []
+
+        for k, v in packages_json.items():
+            print(f"Checking {k}:{v}...")
+            futures.append(executor.submit(check_ipfs_hash_pushed, v))
+
+        # Awaiting for results is blocking
+        print("Awaiting for results...")
+        future_results = [future.result() for future in futures]
+
+        errors = []
+        for future_result in future_results:
+            if not future_result[1]:
```

Comment:
Please comment on the code why index 1 so that it is easier to read without having to navigate to the called fn.